nvs.sh

```diff
@@ -59,7 +59,12 @@ nvs() {
 		fi
 
 		# Download a node binary to use to bootstrap the NVS script.
-		local NODE_ARCH="$(uname -m | sed 's/x86_64/x64/' | sed 's/i686/x86/')"
+		# SmartOS (SunOS) reports `i86pc` which is synonymous with both x86 and x64.
+		local NODE_ARCH="$(uname -m | sed -e 's/x86_64/x64/;s/i86pc/x64/;s/i686/x86/')"
+		# On AIX reports `uname -m` reports the machine ID number of the hardware running the system.
+		if [ "${NVS_OS}" = "aix" ]; then
+			local NODE_ARCH="ppc64"
```

Comment:
nit: `local` is not needed here since the local variable is already declared above.